lib/Parse/ParseType.cpp

```diff
@@ -581,18 +581,47 @@ Parser::parseTypeIdentifier(bool isParsingQualifiedDeclName) {
   }
   SyntaxParsingContext IdentTypeCtxt(SyntaxContext, SyntaxContextKind::Type);
 
-  // Note: these variables are used only when parsing a qualified decl name.
-  // Store the position for backtracking.
-  ParserPosition backtrackingPosition;
-
   ParserStatus Status;
   SmallVector<ComponentIdentTypeRepr *, 4> ComponentsR;
   SourceLoc EndLoc;
   while (true) {
     SourceLoc Loc;
     Identifier Name;
-    // Update backtracking position.
-    backtrackingPosition = getParserPosition();
+
+    // SWIFT_ENABLE_TENSORFLOW
+    // Returns true if the last type identifier component has been parsed.
+    // Lambda is used only when `isParsingQualifiedDeclName` is true.
```

Comment:
Where’s Lambda? Don’t refer to this function as “lambda”, just call it “this function” :)

Reply:
Done in https://github.com/apple/swift/pull/17725/commits/fbbd8dcb5cad897f5fc01436e76ea4ecf11338ef.